Tests/DatadogTests/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsPredicateTests.swift

```diff
@@ -8,16 +8,31 @@ import XCTest
 import Datadog
 
 class UIKitRUMViewsPredicateTests: XCTestCase {
```

Comment:
We could also add test cases to check that some known UIKit classes are filtered out, to catch any regressions of  the internal `isUIKit` with future releases or iOS.

Reply:
+1 to @acgh , we rely on private information (`UIKitCore` is private) and that may break at any time

Reply:
This is already tested on `master`:
https://github.com/DataDog/dd-sdk-ios/blob/4e952d31597cd4a1e1c774c6411e1efb7086dcf4/Tests/DatadogTests/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsPredicateTests.swift#L24

https://github.com/DataDog/dd-sdk-ios/blob/4e952d31597cd4a1e1c774c6411e1efb7086dcf4/Tests/DatadogTests/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsPredicateTests.swift#L11

Reply:
It's already there - just not visible in this diff. We had this test before:
```swift
    func testGivenDefaultPredicate_whenAskingUIKitViewController_itReturnsNoView() {
        // Given
        let predicate = DefaultUIKitRUMViewsPredicate()
        // When
        let uiKitViewController = UIViewController()
        let rumView = predicate.rumView(for: uiKitViewController)
        // Then
        XCTAssertNil(rumView)
    }
```